cmd/glasskube/cmd/describe.go

```diff
@@ -243,8 +290,66 @@ func nameAndDescription(manifest *v1alpha1.PackageManifest) string {
 	return bld.String()
 }
 
+func createOutputStructure(
+	ctx context.Context,
+	pkg *v1alpha1.Package,
+	manifest *v1alpha1.PackageManifest,
+	latestVersion string,
+	repos []v1alpha1.PackageRepository,
+) map[string]interface{} {
+	data := map[string]interface{}{
+		"packageName":      manifest.Name,
+		"shortDescription": manifest.ShortDescription,
+		"latestVersion":    latestVersion,
+		"status":           "Not Installed",
+		"entrypoints":      manifest.Entrypoints,
+		"dependencies":     manifest.Dependencies,
+		"longDescription":  strings.TrimSpace(manifest.LongDescription),
+		"repositories":     repositoriesAsMap(pkg, repos),
+		"references":       referencesAsMap(ctx, pkg, manifest),
+	}
+	if pkg != nil {
+		data["desiredVersion"] = pkg.Spec.PackageInfo.Version
+		data["configuration"] = pkg.Spec.Values
+		data["version"] = pkg.Status.Version
+		data["autoUpdate"] = clientutils.AutoUpdateString(pkg, "Disabled")
```

Comment:
ideally this would be a boolean instead of Enabled/Disabled – however since this PR already has quite a history, I would create a new issue for that, as this is also a quite beginner-friendly task.